src/routes/volume/VolumeBulkActions.js

```diff
@@ -9,33 +9,32 @@ function bulkActions({ selectedRows, engineImages, bulkDeleteVolume, showBulkEng
   const deleteWranElement = (rows) => {
     let workloadResources = []
     let pvResources = []
-    let hasVolumeAttached = false
+    let hasPVRows = []
+    let hasWorkloadsRows = []
 
     rows.forEach((item) => {
       if (item && item.kubernetesStatus && item.kubernetesStatus.pvStatus && item.kubernetesStatus.pvName) {
         pvResources.push((<div style={{ margin: '10px', color: '#b7b7b7' }}>
           <div style={{ paddingLeft: '10px' }}>PV Name: {item.kubernetesStatus.pvName}</div>
           { !item.kubernetesStatus.lastPVCRefAt && item.kubernetesStatus.pvcName ? <div style={{ paddingLeft: '10px' }}>PVC Name: {item.kubernetesStatus.pvcName}</div> : ''}
         </div>))
+        hasPVRows.push(item.name)
       }
       if (item.kubernetesStatus && item.kubernetesStatus.workloadsStatus && item.kubernetesStatus.workloadsStatus.length > 0) {
         workloadResources.push((<div>
           <div style={{ margin: '10px 0px', color: '#b7b7b7' }}>{item.kubernetesStatus.workloadsStatus.map((ele, i) => {
             return (<div key={i} style={{ paddingLeft: '10px' }}>Pod Name: {ele.podName} ({ele.podStatus})</div>)
           })}</div>
         </div>))
-      }
-      if (item.state === 'attached') {
-        hasVolumeAttached = true
+        hasWorkloadsRows.push(item.name)
       }
     })
     return (<div>
-      {hasVolumeAttached ? <div>Some of the selected volumes are attached to a node and may cause errors in any applications using them once they are deleted.</div> : ''}
```

Comment:
Same here. We can keep this message for the case of volumes being attached (even if they don't necessarily have workloads).